tests/models/wav2vec2_conformer/test_modeling_wav2vec2_conformer.py

```diff
@@ -872,6 +919,57 @@ def _load_datasamples(self, num_samples):
 
         return [x["array"] for x in speech_samples]
 
+    def check_inference_pretrained(self, model_id, model, input_name, inputs, features_shape):
+        torch.manual_seed(0)
+        mask_time_indices = _compute_mask_indices(
+            features_shape,
+            model.config.mask_time_prob,
+            model.config.mask_time_length,
+            min_masks=2,
+        )
```

Comment:
It'd be better to move creating the indices to the respective model inference tests and then passing the minimal number of objects to this test: model, model config and inputs then the rest of this function becomes simpler